### PR TITLE
Seong kim

### PR DESCRIPTION
**Pull request by seongck**
MVP + stretch complete

### CODE REVIEW

File: Article/Article.js [hunk 1]
```diff
@@ -3,18 +3,35 @@
...
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = 'Click to Expand'; 
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this)); 
+    // create a reference to the ".close" class
+    this.closeButton = this.domElement.querySelector('.close');
```
Comment (sapinspys) [inline Article/Article.js]: Rename to `this.deleteButton` for better readability

File: Article/Article.js [hunk 2]
```diff
@@ -3,18 +3,35 @@
...
+    // Set a click handler on the closeButton reference, calling the
+    // deleteArticle method
+    this.closeButton.addEventListener('click', this.deleteArticle.bind(this));
   }
 
   expandArticle() {
```
Comment (sapinspys) [inline Article/Article.js]: Good work, can simplify to:

```
  expandArticle() {
    // Using our reference to the domElement, toggle a class to expand or hide the article.
    this.domElement.classList.toggle('article-open');

    this.expandButton.innerText === 'Click to Expand')
      ? this.expandButton.innerText = 'Click to Close'
      : this.expandButton.innerText = 'Click to Expand';
  }
```

File: CSS/index.css [hunk 3]
```diff
@@ -35,6 +35,109 @@
...
+  width: 350px;
+  height: 100vh;
+  /* This give the height 100% of the screen or, '100% of View Height' */
+  background-color: #81C784;
+  border: 1px solid #004D40;
+  border-left: none;
+  border-bottom: none;
+  z-index: 2;
+  /* This set the layer level of the element, it has precendence over initial level of 1*/
+  transform: translateX(-100%);
```
Comment (sapinspys) [inline CSS/index.css]: Clean and simple CSS styling, very functional

**Comment by seongck** [comment]
Thanks for the feedback!

**Pull request merged**